patches/chrome-android-BUILD.gn.patch

```diff
@@ -19,15 +19,23 @@ index ac52a743f73b7109f57df8f677518ca717812ba7..ff92fd9e070e290566ebfb401afc3b3a
      "//chrome/android/webapk/libs/common:splash_resources",
      "//chrome/app:java_strings_grd",
      "//chrome/browser/ui/android/widget:ui_widget_java_resources",
-@@ -543,6 +544,7 @@ java_group("chrome_all_java") {
+@@ -276,6 +277,7 @@ android_library("chrome_java") {
+     "$google_play_services_package:google_play_services_gcm_java",
+     "$google_play_services_package:google_play_services_iid_java",
+     "$google_play_services_package:google_play_services_tasks_java",
++    "$google_play_services_package:google_play_services_safetynet_java",
+     "//base:base_java",
+     "//base:jni_java",
+     "//chrome/android/features/keyboard_accessory:public_java",
+@@ -543,6 +545,7 @@ java_group("chrome_all_java") {
  group("jni_headers") {
    public_deps = [
      ":chrome_jni_headers",
 +    "//brave/build/android:jni_headers",
```

Comment:
this should go in `public_deps += [ "//brave/build/android:jni_headers" ]`

Reply:
the change wasn't added in that PR, I think we had a discussion about that and decided to add it like this and just add jni_headers to brave's file in the future.

Reply:
well, the reason for using `public_deps += [ "//brave/build/android:jni_headers" ]` instead of adding inline in the existing list is to reduce the potential for conflicts when upstream adds items to the list